src/Client.js

```diff
@@ -200,9 +200,8 @@ Client.prototype._performRequest = function (action, path, data, query) {
     rq.query(query);
   }
 
-  if (data) {
-    rq.send(data);
-  }
+  rq.type('json');
+  rq.send(JSON.stringify(data));
```

Comment:
I think we need to test for `undefined`. The previous if statement prevented `undefined` to be sent to the server.